htdocs/checkentropy.php

```diff
@@ -66,5 +67,12 @@ function checkEntropyJSON($password_base64)
     $newpass = htmlspecialchars($_GET["password"]);
     checkEntropyJSON($newpass);
 }
+if(isset($change_custompwdfield)) {
+    foreach ($change_custompwdfield as $custompwdfield) {
+        if ( isset($custompwdfield['pwd_policy_config']) && isset($custompwdfield['pwd_policy_config']['pwd_display_entropy']) && $custompwdfield['pwd_policy_config']['pwd_display_entropy']) {
+            checkEntropyJSON($newpass);
+        }
+    }
+}
```

Comment:
Actually, this check is not correct, because as soon as one pwd_display_entropy has been enabled for any custompwdfield, the entropy check will be done.

Anyway, this is not a big deal, because I don't think the initial check: `if( isset($pwd_display_entropy) && $pwd_display_entropy == true )` is really necessary. This check is already done at the frontend side

I will simply remove it.